packages/react/src/components/InAppMessaging/BannerMessage/styles.css

```diff
@@ -0,0 +1,84 @@
+.body {
+  color: #000;
+  font-size: 16px;
+  font-weight: 400;
+  text-align: left;
+}
+.button {
+  background-color: #e8e8e8;
+  border-radius: 5;
+  border-style: none;
+  width: 100%;
+}
+.buttonsContainer {
+  flex-direction: row;
+  justify-content: space-evenly;
+  padding: S4px;
+}
+.buttonText {
+  font-size: 16px;
+  font-weight: 400;
+  text-align: center;
+}
+.container {
+  background-color: #fff;
+  box-shadow: 2px 2px 9px 3px rgb(0, 0, 0, 0.1);
+  flex-direction: column;
+  margin: 16px;
+  padding: 12px;
+  position: fixed;
+  right: 0;
+  row-gap: 0;
+}
+@media only screen and (min-width: 600px) {
+  .container {
+    max-width: 50%;
+    min-width: 400px;
+  }
+}
+@media only screen and (max-width: 600px) {
+  .container {
+    width: calc(100% - 32px);
+  }
+}
+.contentContainer {
+  flex-direction: row;
+  margin-bottom: 6px;
+}
+.icon {
+  color: #000;
+  gap: initial;
+  justify-content: flex-end;
+}
+.image {
+  object-fit: contain;
+}
+.imageContainer {
+  justify-content: center;
+  min-width: 75px;
+  max-width: 100px;
+  padding: 5px;
+  width: 15%;
+}
+.textContainer {
+  flex: 1;
+  flex-direction: column;
+  gap: initial;
+  justify-content: center;
+}
+.wrapper {
+  align-self: top;
+  background-color: transparent;
+  display: flex;
+  flex: 1;
+}
+.TOP_BANNER {
```

Comment:
Similarly, we should try to avoid `UPPER_SNAKE_CASE` for CSS class names. But I think this all needs to move to the `@aws-amplify/ui` package anyway as @zchenwei alluded to above